package.json

```diff
@@ -39,13 +39,13 @@
     "@kubernetes/client-node": "^0.17.0",
     "@snyk/dep-graph": "^1.30.0",
     "async": "^3.2.4",
-    "aws-sdk": "^2.1200.0",
+    "aws-sdk": "^2.1276.0",
     "bunyan": "^1.8.15",
     "child-process-promise": "^2.2.1",
     "fs-extra": "^10.1.0",
     "lru-cache": "^6.0.0",
-    "minipass": "3.2.1",
-    "needle": "^3.0.0",
+    "minipass": "^4.0.0",
```

Comment:
Do we still need the `minipass`? It was a workaround before? If we remove it, will k-m fail? 😄

Reply:
I can't remember any context around this, what was it a workaround for?

Reply:
https://github.com/snyk/kubernetes-monitor/commit/77a36c13d0c77a49e8fd2e92f36fbda2bd955a70
https://snyk.slack.com/archives/C022DN6CMHS/p1657544371962219